### PR TITLE
Added 2Gb swap creation to the setup stage

### PR DESCRIPTION
**Pull request by dmishh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Set up local ruby gemset and gems from Gemfile `bundle install`
 ### Server
 
 1. Set up an IP address, roles and packages in `nodes.yml`. You can use an example for this `cp nodes.yml.example nodes.yml`. It contains all the options available.
-1. Run setup stage to set up deployer user and automatic security upgrades `STAGE=setup ruby provision.rb`
+1. Run setup stage to set up deployer user, automatic security upgrades and enable 2 Gb swap `STAGE=setup ruby provision.rb`. You can skip swap creation just by adding `NO_SWAP=true`
```
Comment (rilian) [inline README.md]: make it a list, not text

**Comment by rilian** [comment]
generally good

File: README.md (hunk 1 above)
Comment (dmishh) [inline README.md]: But why? This is related to the setup stage. It's not a separate stuff

Comment (rilian) [inline README.md]: * set up deployer user
* automatic security upgrades
* enable 2 Gb swap

these are different things and i think this is important to extract from a big text (no one really reads texts) .......

Comment (rilian) [inline README.md]: let's merge and improve this later

Comment (dmishh) [inline README.md]: Ok

**Pull request merged**